tests/virt_autotest/libvirt_virtual_network_init.pm

```diff
@@ -69,9 +69,14 @@ sub run_test {
         save_guest_ip($guest, name => "br123");
         my $mode = is_sle('=11-sp4') ? '' : '-f';
         exec_and_insert_password("ssh-copy-id -o StrictHostKeyChecking=no $mode root\@$guest");
+        check_guest_module($guest, module => "acpiphp");
         #Prepare the new guest network interface files for libvirt virtual network
         assert_script_run("ssh root\@$guest 'cd /etc/sysconfig/network/; cp ifcfg-eth0 ifcfg-eth1; cp ifcfg-eth0 ifcfg-eth2; cp ifcfg-eth0 ifcfg-eth3; cp ifcfg-eth0 ifcfg-eth4; cp ifcfg-eth0 ifcfg-eth5; cp ifcfg-eth0 ifcfg-eth6'");
-        assert_script_run("ssh root\@$guest 'rcnetwork restart'", 60);
+        if (($guest =~ m/sles11/i) || ($guest =~ m/sles-11/i)) {
```

Comment:
Cosmetic suggestion:

    if ($guest =~ m/sles-?11/i) {

Reply:
Cool! it was so helpful.

Reply:
@grisu48 had updated code as your suggestion. 
Also, updated VR:
http://10.67.19.82/tests/639
Please help me make sure if there was any comments for this PR